src/components/illustrations/XPIcon.tsx

```diff
@@ -1,57 +1,85 @@
-import { useTheme } from '@mui/material';
-import React from 'react';
+import type { BoxProps } from '@mui/material';
+import { alpha, Box, styled, useTheme } from '@mui/material';
+
+interface XPIconBackgroundProps extends BoxProps {
+  bgColor?: string;
+  size?: number;
+}
+
+export const XPIconBackground = styled(Box, {
```

Comment:
Needs to be refactored, IconLVL IconXP and IconRank are duplication of this file https://github.com/jumperexchange/jumper-exchange/pull/1474/files#diff-a3f45a5f6e008c6edd378f7ee1b55b7f0a52dcfcd5285735613461c2d3a4c114R32

Reply:
good catch, thanks!